rpc/handlers.go

```diff
@@ -1213,10 +1214,10 @@ func (h *Handler) TraceBlockTransactions(blockHash felt.Felt) ([]TracedBlockTran
 		return nil, ErrBlockNotFound
 	}
 
-	return h.traceBlockTransactions(block)
+	return h.traceBlockTransactions(block, len(block.Transactions))
 }
 
-func (h *Handler) traceBlockTransactions(block *core.Block) ([]TracedBlockTransaction, *jsonrpc.Error) {
+func (h *Handler) traceBlockTransactions(block *core.Block, maxTransactionIdx int) ([]TracedBlockTransaction, *jsonrpc.Error) {
```

Comment:
something like `numTxns` sounds better to me, instead of `maxTransactionIdx`. But up to you.

Reply:
Yep, renamed